src/main/main.cpp

```diff
@@ -30,6 +30,7 @@
 #include <etl/ReportingETL.h>
 #include <log/Logger.h>
 #include <rpc/Counters.h>
+#include <rpc/RPCEngine.h>
```

Comment:
Why we add this header here?

Reply:
For `RPCEngine::make_RPCEngine()` factory.